### DISCUSSION TITLE
麦克风使用 | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1304] [Q&A]
# 麦克风使用 | Seeed Studio Wiki

This article describes how to use the microphone on the XIAO ESP32S3 Sense expansion board.

https://wiki.seeedstudio.com/cn/xiao_esp32s3_sense_mic/

**Comment by halleyhaoyu** [comment]
Compilation error: I2S.h: No such file or directory

**Reply by halleyhaoyu** [reply]
When ESP32 development board update to version 3.0.0, this code not compatible anymore. But this code still work with version 2.0.X.
